<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="6" passed="3" failed="2" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[Added assertion for method]]>
    </line>
    <line>
      <![CDATA[Added assertion2 for method]]>
    </line>
  </reporter-output>
  <suite started-at="2020-01-27T12:52:49Z" name="DataProvider and listener" finished-at="2020-01-27T12:52:49Z" duration-ms="41">
    <groups>
      <group name="first">
        <method signature="CustomLoggingTest.testMethodTwo()" name="testMethodTwo" class="CustomLoggingTest"/>
        <method signature="CustomLoggingTest.testMethodThree()" name="testMethodThree" class="CustomLoggingTest"/>
        <method signature="CustomLoggingTest.testMethodOne()" name="testMethodOne" class="CustomLoggingTest"/>
        <method signature="CustomLoggingTest.MethodSix()" name="MethodSix" class="CustomLoggingTest"/>
      </group>
      <group name="second">
        <method signature="CustomLoggingTest.MethodSix()" name="MethodSix" class="CustomLoggingTest"/>
      </group>
    </groups>
    <test started-at="2020-01-27T12:52:49Z" name="Check LoggingTests" finished-at="2020-01-27T12:52:49Z" duration-ms="17">
      <class name="CustomLoggingTest">
        <test-method signature="testMethodTwo()" started-at="2020-01-27T12:52:49Z" name="testMethodTwo" finished-at="2020-01-27T12:52:49Z" duration-ms="6" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<true> but was:<false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:480)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at org.testng.Assert.assertTrue(Assert.java:47)
	at CustomLoggingTest.testMethodTwo(CustomLoggingTest.java:18)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="MethodSix()" started-at="2020-01-27T12:52:49Z" name="MethodSix" finished-at="2020-01-27T12:52:49Z" duration-ms="0" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<true> but was:<false>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:480)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at org.testng.Assert.assertTrue(Assert.java:47)
	at CustomLoggingTest.MethodSix(CustomLoggingTest.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method signature="testMethodThree()" started-at="2020-01-27T12:52:49Z" name="testMethodThree" finished-at="2020-01-27T12:52:49Z" depends-on-methods="CustomLoggingTest.testMethodTwo" duration-ms="1" status="SKIP">
        </test-method>
        <test-method signature="testMethodOne()" started-at="2020-01-27T12:52:49Z" name="testMethodOne" finished-at="2020-01-27T12:52:49Z" duration-ms="1" status="PASS">
        </test-method>
      </class>
    </test>
    <test started-at="2020-01-27T12:52:49Z" name="Test2" finished-at="2020-01-27T12:52:49Z" duration-ms="10">
      <class name="TestWithDataProvider">
        <test-method signature="printData(java.lang.String, java.lang.String, int)" started-at="2020-01-27T12:52:49Z" name="printData" finished-at="2020-01-27T12:52:49Z" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[myTestFirstName]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[firstValue]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[15]]>
              </value>
            </param>
          </params>
        </test-method>
        <test-method signature="printData(java.lang.String, java.lang.String, int)" started-at="2020-01-27T12:52:49Z" name="printData" finished-at="2020-01-27T12:52:49Z" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[myTestLastName]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secondValue]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[45]]>
              </value>
            </param>
          </params>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
